### PR TITLE
Redesign of Signup/Login Pages

### PR DESCRIPTION
**Pull request by jb162**
Redesigned the signup and login pages, including the AuthHeader, according to mockups. Added logo and buttons to AuthHeader.

### What this PR does (required):
- redesigned login and signup pages according to mockups, (as close as possible)
- redesigned AuthHeader (logo, "become a sitter", login & signup buttons, box-shadow)

### Screenshots / Videos (front-end only):
- Login Page Screenshot
login-screenshot
- Signup Page Screenshot
signup-screenshot

### Any information needed to test this feature (required):
- Signup: create a new account using the Signup page
- Login: login to your new account using the Login page

### Any issues with the current functionality (optional):
- I am unsure of the exact style specifications defined by the mockup. I tried to create the redesign as close as I could to the mockup but a style guide would be very helpful.
- replaced TextField component with FormControl, FormLabel, OutlinedInput, and FormHelperText to achieve the desired styling
- this may cause problems if other code references these components by TextField instead of id or className
Screen Shot 2021-11-08 at 4 16 55 PM

### Updates (Tuesday, November 9th, 2021)
-replaced sample.env with .env file containing MONGO_URI
-added config.js file to pass MONGO_URI environment variable (uses dotenv) to db.js as an import
-added alt attribute to logo image in AuthHeader.tsx
-combined MUI Button and react-router Link components into one Button component in AuthHeader.tsx
-changed uppercase text to lowercase, then used textTransform to change text to uppercase (AuthHeader, LoginForm, SignUpForm)
-moved CSS properties from useStyles to Box component directly (AuthHeader, Login, SignUp)
-added red color (#f04040) to MUI theme as theme.palette.primary.main
-added green color (#30E664) to MUI theme as theme.typography.button.hoverColor

### CODE REVIEW

**Review by ezeilo-su — CHANGES_REQUESTED** [review]
Looks good! Please checkout the comments below and make the necessary changes.
Happy coding!

File: client/src/components/AuthHeader/AuthHeader.tsx [hunk 1]
```diff
@@ -15,12 +16,18 @@ const AuthHeader = ({ linkTo, asideText, btnText }: Props): JSX.Element => {
 
   return (
     <Box p={1} className={classes.authHeader}>
-      <Typography className={classes.accAside}>{asideText}</Typography>
-      <Link to={linkTo} className={classes.link}>
-        <Button color="inherit" className={classes.accBtn} variant="contained">
-          {btnText}
-        </Button>
-      </Link>
+      <img className={classes.headerLogo} src={logo} />
```
Comment (rajivtitus) [inline client/src/components/AuthHeader/AuthHeader.tsx:19]: It is a good practice to add 'alt' attributes to images :)

File: client/src/components/AuthHeader/AuthHeader.tsx [hunk 2]
```diff
@@ -15,12 +16,18 @@ const AuthHeader = ({ linkTo, asideText, btnText }: Props): JSX.Element => {
...
+      <Box className={classes.buttonContainer}>
+        <Typography className={classes.headerText}>BECOME A SITTER</Typography>
+
+        <Link to={'/login'} className={classes.link}>
+          <Button className={classes.loginBtn}>LOGIN</Button>
```
Comment (rajivtitus) [inline client/src/components/AuthHeader/AuthHeader.tsx:24]: The MUI Button and react-router Link can be combined:

`
<Button component={Link} to="/login" color="primary">login</Button>
`

Ditto throughout!

Comment (rajivtitus) [inline client/src/components/AuthHeader/AuthHeader.tsx:24]: Please avoid using CAPITALCASE for text, some screen readers interpret this text as abbreviations, you can resort to using a CSS class to transform text instead.

File: client/src/components/AuthHeader/useStyles.ts [hunk 3]
```diff
@@ -3,10 +3,13 @@ import { makeStyles } from '@material-ui/core/styles';
 const useStyles = makeStyles((theme) => ({
   authHeader: {
     alignSelf: 'flex-end',
-    justifyContent: 'flex-end',
+    justifyContent: 'space-between',
```
Comment (rajivtitus) [inline client/src/components/AuthHeader/useStyles.ts:6]: Most of these styles can be applied to the Box component directly

Eg: 
`
<Box display="flex" alignItems="center".....></Box>
`

File: client/src/components/AuthHeader/useStyles.ts [hunk 4]
```diff
@@ -30,6 +33,56 @@ const useStyles = makeStyles((theme) => ({
...
+    borderRadius: '2',
+    fontSize: 12,
+    backgroundColor: 'white',
+    color: '#f04040',
+    fontWeight: 'bold',
+    border: '1px solid #f04040',
+    marginRight: 10,
+    transition: '1s',
+    '&:hover': {
+      backgroundColor: '#f04040',
```
Comment (rajivtitus) [inline client/src/components/AuthHeader/useStyles.ts:59]: You can set colours within MUI's theme and use the theme to apply them  which ensures consistency across the app :)

File: client/src/pages/Login/LoginForm/LoginForm.tsx [hunk 5]
```diff
@@ -46,47 +47,43 @@ export default function Login({ handleSubmit }: Props): JSX.Element {
...
-            type="password"
-            autoComplete="current-password"
-            helperText={touched.password ? errors.password : ''}
-            error={touched.password && Boolean(errors.password)}
-            value={values.password}
-            onChange={handleChange}
-          />
+          <FormControl margin="normal" fullWidth>
+            <FormLabel>
+              <Typography className={classes.label}>EMAIL ADDRESS</Typography>
```
Comment (rajivtitus) [inline client/src/pages/Login/LoginForm/LoginForm.tsx:52]: Same as comment above for CAPITALCASE text

File: client/src/pages/Login/useStyles.ts [hunk 6]
```diff
@@ -13,15 +13,31 @@ const useStyles = makeStyles(() => ({
...
   welcome: {
-    fontSize: 26,
+    textAlign: 'center',
+    fontSize: 42,
     paddingBottom: 20,
     color: '#000000',
     fontWeight: 700,
     fontFamily: "'Open Sans'",
   },
+  loginForm: {
```
Comment (rajivtitus) [inline client/src/pages/Login/useStyles.ts:28]: Most of the styles within this class can be directly applied to the Box component 😄

File: client/src/pages/SignUp/useStyles.ts [hunk 7]
```diff
@@ -13,15 +13,31 @@ const useStyles = makeStyles(() => ({
...
-    fontSize: 26,
-    paddingBottom: 20,
+    textAlign: 'center',
+    fontSize: 42,
+    paddingBottom: 30,
     color: '#000000',
     fontWeight: 700,
     fontFamily: "'Open Sans'",
   },
+  signupForm: {
```
Comment (rajivtitus) [inline client/src/pages/SignUp/useStyles.ts:28]: Most of the styles within this class can be applied to the Box component directly :)

**Review by rajivtitus — COMMENTED** [review]
UI looks good, nice work here! Please have a look and address comment from Sunday and myself. Thank you!

File: client/src/components/AuthHeader/useStyles.ts [hunk 8]
```diff
@@ -1,16 +1,7 @@
...
-    alignSelf: 'flex-end',
-    justifyContent: 'space-between',
-    alignItems: 'center',
-    display: 'flex',
-    flexWrap: 'wrap',
-    width: '100%',
-    backgroundColor: 'white',
-    boxShadow: 'rgba(100, 100, 111, 0.2) 0px 7px 29px 0px',
-  },
+  authHeader: {},
```
Comment (rajivtitus) [inline client/src/components/AuthHeader/useStyles.ts:4]: Please remove empty the empty CSS class here :)

File: client/src/pages/Login/useStyles.ts [hunk 9]
```diff
@@ -7,23 +7,13 @@ const useStyles = makeStyles(() => ({
...
-    display: 'flex',
-    alignItems: 'flex-start',
-    justifyContent: 'space-between',
-    flexDirection: 'column',
-    minHeight: '100vh',
-    minWidth: '100vw',
-    paddingTop: 0,
-    backgroundColor: 'white',
-  },
+  authWrapper: {},
```
Comment (rajivtitus) [inline client/src/pages/Login/useStyles.ts:10]: Please remove empty class

File: client/src/pages/SignUp/useStyles.ts [hunk 10]
```diff
@@ -7,23 +7,13 @@ const useStyles = makeStyles(() => ({
...
-    display: 'flex',
-    alignItems: 'flex-start',
-    justifyContent: 'space-between',
-    flexDirection: 'column',
-    minHeight: '100vh',
-    minWidth: '100vw',
-    paddingTop: 0,
-    backgroundColor: 'white',
-  },
+  authWrapper: {},
```
Comment (rajivtitus) [inline client/src/pages/SignUp/useStyles.ts:10]: Please remove

**Review by rajivtitus — APPROVED** [review]
Great work making those changes 💯 
Just some minor cleanup to do prior to merging 👍🏽

**Pull request merged**